extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -336,6 +348,13 @@ public void run()
             TimeUnit.MILLISECONDS
         );
 
+        metricEmittingExec.scheduleAtFixedRate(
+            computeAndEmitLag(taskClient),
+            ioConfig.getStartDelay().getMillis() + 10000, // wait for tasks to start up
```

Comment:
I am not sure what you mean by using `druid.monitoring.emissionPeriod` as initial delay, it gives emission period and how is it would relate to initial delay. This code is run when supervisor starts therefore it will emit only is any supervisor is running.